docs/user/configuring_the_client.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Configuring the Client
+---
+
+## Environmental variables
+
+Please note that the underlying libraries that the Rucio client uses (e.g. GFAL) may be further affected by environmental variables not mentioned in this page.
+
+### ATLAS_SITE_NAME
```

Comment:
We should deprecate this - opened this issue to track this: https://github.com/rucio/rucio/issues/7022